docs/configuration.rst

```diff
@@ -292,7 +292,9 @@ The following environment variables for the tracer are supported:
      description: |
          A map of case-insensitive header keys to tag names. Automatically applies matching header values as tags on root spans.
 
-         For example, ``User-Agent:http.useragent,content-type:http.content_type``.
+         For example, ``User-Agent:http.useragent,content-type:http.content_type``. In this example the will set ``http.useragent`` and ``http.content_type`` tags 
```

Comment:
With this PR, I think it'd be good to clarify that both formats would be supported in the configuration doc:

To customize the name of the span tag, use this format:
```
User-Agent:http.useragent,content-type:http.content_type
```


Otherwise, let the python tracer come up with a default span tag name for the header:
```
User-Agent,content-type
```